app/common.py

```diff
@@ -27,6 +27,13 @@ class Channel(Enum):
     TELEGRAM = "Telegram"
 
 
+class Category(Enum):

```

Comment:
ALWAYS_FREE is semantically the same as OfferDuration.ALWAYS.

Since we have scrapers that exclusively track offers that are "always free", I think that this is still "valid". Wdyt?

Reply:
Do we want to duplicate all scrapers for games that are free forever? I thought about the categories like a list of things people could exclude offers for. ALWAYS_FREE is a reason to exclude an offer, at the moment offers can't be filtered by OfferDuration.

PS.: ALWAYS_FREE also includes offers which are valid for over 100 years, but that's a technicality.

Reply:
Why would we want to duplicate scrapers? I think we have to distuinguish "getting the data" and "presenting the data" here. For the first one, I think it is enough to store the information once (here in "OfferDuration") instead of duplicating it into "Category".

If it's about including or excluding offers, I think it would be fine if a user could exclude based on
- "No Demos" (Category.DEMO)
- "No Cheap stuff" (Category.CHEAP)
- "No temporary offers" (OfferDuration.TEMPORARY)
- "No always free games" (OfferDuration.ALWAYS)

Probably this needs a bit of thinking how to present the options to the user (for the Telegram bot) and how to split the RSS feeds as to not generate hundreds of them multiplying out the options.

But I think this something to flesh out in a future PR. Let's focus on the "data gathering" side in this one to not mix up too much.

Reply:
> Why would we want to duplicate scrapers?

I don't want that, I was referring to the way it's done currently with  `gog_games` and `gog_games_alwaysfree` (if I understand that right).

> If it's about including or excluding offers, I think it would be fine if a user could exclude based on
> 
> * "No Demos" (Category.DEMO)
> * "No Cheap stuff" (Category.CHEAP)
> * "No temporary offers" (OfferDuration.TEMPORARY)
> * "No always free games" (OfferDuration.ALWAYS)

That's not a simple filter on one field but works since all the data is in one object anyways.

Reply:
I removed `ALWAYS_FREE`. I suggest getting the durations for e.g. Humble Bundle Offers and setting `OfferDuration.ALWAYS`in another PR.